flasksrc/quiz.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, request
+from flask_restful import Resource, Api, abort
+
+from flasksrc.db import get_db
+
+class ToyNetQuiz(Resource):
+    def get(self, quiz_id):
+        db = get_db()
+        error = None
+
+        rows_question = db.execute(
+            'SELECT qq.question_id, qq.question, qq.answer'
+            ' FROM toynet_quiz_questions AS qq'
+            ' WHERE qq.quiz_id = (?) ',
+            (str(quiz_id),)
+        ).fetchall()
+
+        rows_option = db.execute(
+            'SELECT qo.question_id, qo.option'
+            ' FROM toynet_quiz_options AS qo'
+            ' WHERE qo.quiz_id = (?)'
+            ' ORDER BY qo.question_id, qo.option_id',
+            (str(quiz_id),)
+        ).fetchall()
+
+        if not len(rows_question):
```

Comment:
let's also check that:
(1) it has options
(2) the answer # is less than the length of these options

The error code would be 500 (internal server error) since if the quiz question exists, then so should the options and answers so it's a valid request that the server failed to respond to.